news/2019-03-10-project-loved-week-of-march-10th.md

```diff
@@ -0,0 +1,207 @@
+
+A map almost lost to time resurfaces to participate in this week's Project Loved voting! Some may recognize this song from a previous Loved map by [RyuK](https://osu.ppy.sh/users/6304246), however this mapset by Mordred is considered by many to be the first successful and widely accepted rendition of Seiryu's AO-Infinity. This set comes with two difficulties: Vill's EXHAUST at 6.52 stars, and Mordred's INFINITY at 7.09 stars. Each difficulty does a great job at emphasizing the streamy and jumpy nature of the song and creates a perfect environment for players looking for a challenge on some good tunes.
+
+[![](/wiki/shared/news/2019-03-10-project-loved-week-of-march-10th/osu/error.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[Aimiya Zero - -ERROR](https://osu.ppy.sh/beatmapsets/426602#osu) by [Yusomi](https://osu.ppy.sh/users/4174940)  
+*written by [BeasttrollMC](https://osu.ppy.sh/users/3171691)*
+
+Known for her high difficulty and jump-based maps, Yusomi returns again with her take on Aimiya Zero's "-ERROR". The song tells a story of a character drowning from their emotions after a heartbreaking separation. Although trivial, the mapset does a clean job of simulating a similar experience. Instead of emotions, any player who attempts this map will find themselves drowning in a barrage of increasingly difficult 200 BPM jumps and bursts. I think it's a fitting theme for the song. The map itself has amassed over 110 favorites since its submission in March of 2016, and is has been recognized by many top players for its difficulty. However, its survival in the voting depends on you. Check out the map and come back to decide its fate in Project Loved!
+
+[![](/wiki/shared/news/2019-03-10-project-loved-week-of-march-10th/osu/night-club-junkie.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[YUC'e - Night Club Junkie](https://osu.ppy.sh/beatmapsets/756066#osu) by [Nathan](https://osu.ppy.sh/users/4785223) and [fanzhen0019](https://osu.ppy.sh/users/418699)  
+*written by [DigitalHypno](https://osu.ppy.sh/users/4384207)*
+
+Gather 'round the nightclub, everyone! Rumor has it that there's a spicy new YUC'e song trying to make its way into the Loved category. The mapset is hosted by Nathan and has one difficulty from him and a guest difficulty from fanzhen0019. Seems promising—let's check out the details. Both difficulties are centered around slider overlap reading, each with their own overlap gimmick that's executed throughout the entire map. Well, give me your thoughts! Check out the map for yourself, then come on back and drop your vote on whether or not this mapset should be brought to the Loved category.
+
+[![](/wiki/shared/news/2019-03-10-project-loved-week-of-march-10th/osu/60th-summer-of-love.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[forest - 60th Summer of Love](https://osu.ppy.sh/beatmapsets/803415#osu) by [Fushimi Rio](https://osu.ppy.sh/users/1205412)  
```

Comment:
I think `Forest` (upper) is the correct one as i [googled it](https://utaite.fandom.com/wiki/Usa).

Reply:
@dorothy3242 could you provide a source for that, I'm assuming yours is correct

Reply:
dorothy says
http://forest.lovepop.jp/
https://colorful-bouquet.tumblr.com/
https://hana-forest.booth.pm/items/580067

so lowercase is OK :+1: